sceptre/plan/actions.py

```diff
@@ -1055,6 +1056,7 @@ def drift_show(self) -> Tuple[str, dict]:
         else:
             raise Exception("Not expected to be reachable")
 
+        response = self._filter_drifts(response, drifted)
```

Comment:
Why not only apply this when `drifted`?

```suggestion
        if drifted:
            response = self._filter_drifts(response, drifted)
```